creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -979,32 +979,25 @@ class SnapbackSM {
   /**
    * Monitor an ongoing sync operation for a given secondaryUrl and user wallet
    * Return boolean indicating if an additional sync is required
-   *
-   * Polls secondary for MaxSyncMonitoringDurationInMs
    */
   async additionalSyncIsRequired (userWallet, primaryClockValue = -1, secondaryUrl, syncType) {
     const logMsgString = `additionalSyncIsRequired (${syncType}): wallet ${userWallet} secondary ${secondaryUrl} primaryClock ${primaryClockValue}`
 
-    // Define axios request object for secondary clock status request
-    const clockStatusRequestParams = {
-      method: 'get',
-      baseURL: secondaryUrl,
-      url: `/users/clock_status/${userWallet}`,
-      responseType: 'json'
-    }
-
     const startTimeMs = Date.now()
     const maxMonitoringTimeMs = startTimeMs + MaxSyncMonitoringDurationInMs
 
-    let secondaryCaughtUp = false
-    let secondaryProgressed = false
+    /**
+     * Poll secondary for sync completion, up to `maxMonitoringTimeMs`
+     */
+
+    let secondaryCaughtUpToPrimary = false
     let initialSecondaryClock = null
     let finalSecondaryClock = null
+
     while (Date.now() < maxMonitoringTimeMs) {
       try {
-        const clockStatusResp = await axios(clockStatusRequestParams)
-        const { clockValue: secondaryClockValue } = clockStatusResp.data.data
-        this.log(`${logMsgString} secondaryClock ${secondaryClockValue}`)
+        const secondaryClockValue = await CreatorNode.getClockValue(secondaryUrl, userWallet)
```

Comment:
with my rate limit PR changes, you probably want to add in the spID, signature, and timestamp here too as so that this SP doesn't get rate limited

Reply:
https://github.com/AudiusProject/audius-protocol/pull/1670/files#diff-62e7d629775fe1c67406f64ab0a79b6aa6ed5bfb98fe5ed4e2048dc45bf4ed88R551-R556

Reply:
yep, i'll handle this when rebasing on top of yours